pepper-apis/dsm-server/src/resources/config/application.conf.default

```diff
@@ -0,0 +1,251 @@
+  ]
+
+  // Endpoint for the Auth0 Management API for the auth0.account
+  mgtApiUrl = "URL-TO-AUTH0-MANAGEMENT-ENDPOINT"
+  // Client ID for the client used to access the Auth0 management API
+  mgtKey = "AUTH0-CLIENT-ID"
+  // Client Secret for the client used to access the Auth0 management API
+  mgtSecret = "AUTH0-CLIENT-SERCRET"
+}
+
+// ####
+// Email config
+// ####
+email {
+  key = "SG.SENDGRID-API-KEY"
+  notifications = "[]"
+  frontendUrl = "http://localhost:4200"
+  // Crontab-formatted specification for when to send GP emails 
+  cron_expression_GP_notification = "0 30 11 ? * MON-FRI" 
+  gp_recipient = "local-dev@domain.example"
+  clientSettings = """{
+    "sendGridFrom": "local-dev@domain.example",
+    "sendGridFromName": "DSM Local"
+  }"""
+  
+  reminderNotifications = """[
+    {
+      "reason": "PARTICIPANT_REMINDER",
+      "reminders": [
+        {
+          "adminRecipient": "",
+          "hours": "1",
+          "sendGridTemplate": "SENDGRID-TEMPLATE-UUID"
+        }
+      ]
+    }
+  ]"""
+}
+
+// ----
+// Quartz Jobs
+// ----
+quartz {
+  enableJobs = false
+  ddpRequestJob_jobIntervalInSeconds = 120
+  notificationJob_jobIntervalInSeconds = 60
+  labelCreationJob_jobIntervalInSeconds = 60
+  ddp_event_trigger_cron_expression = "0 0/2 8-18 * * ?"
+  externalShipper_cron_expression = "0 0/5 0 ? * MON-FRI"
+  externalShipper_cron_expression_additional = "0 0 2 ? * MON-FRI"
+  shipmentStatusJob_cron_expression = "0 0/5 * ? * MON-FRI"
+}
+
+elasticSearch {
+  url = "http://localhost:9200"
+}
+
+// The 'tokenSecret' key appears to be only used in tests?
+ddp = """[
+    {
+        "easyPostApiKey": "EASYPOST-API-KEY",
+        "instanceName": "an-instance",
+        "tokenSecret": "shhhhh"
+    }
+]"""
+
+gbf = """[ ]"""
+
+externalShipper = """[ ]"""
+
+pubsub {
+  //
+  // Set to `true` if DSM should use a local Pub/Sub host.
+  //
+  // This flag is used to trigger a change in behaviors in order to ease local 
+  // development work using Google's Pub/Sub Emulator.
+  //   * DSM will attempt to connect to the Pub/Sub host at `pubsub.host`
+  //   * Use of credentials will be disabled when communication with Pub/Sub
+  //   * Pub/Sub Topics will now attempt to be created on launch
+  //   * Pub/Sub Subscriptions will now attempt to be created on launch
+  //
+  emulator = true
+  projectId = "google-project-id"
+
+  // The Pub/Sub host to connect to.
+  // This value has no effect unless `pubsub.local` is
+  //  set to `true`.
+  host = "127.0.0.1:8085"
+
+  topic = "dsx-local-dsm-test-results"
+  subscription = "dsx-local-dsm-test-results-sub"
+
+  dsm_to_dss_topic = "dsx-local-dss-tasks"
+  dss_tasks_topic = "dsx-local-dss-tasks"
+
+  dss_to_dsm_topic = "dsx-local-dss-task-results"
+  dss_to_dsm_subscription = "dsx-local-dss-task-results-sub"
+
+  dsm_tasks_topic = "dsx-local-dsm_tasks_topic"
+  dsm_tasks_subscription = "dsx-local-dsm-tasks-sub"
+
+  dsm_to_mercury_topic = "dsx-local-mercury-pepper"
```

Comment:
Prefixed with `dsx-local` to make it easy to identify topics/subscriptions if they are accidentally created outside of a local dev environment.